### ISSUE TITLE
Provide RSS feed

### ISSUE THREAD

**Issue by sukunrt** [libp2p/blog#88]
Rss doesn't work on blog.libp2p.io currently. There's blog.libp2p.io/index.xml but the links there are incorrect. 

```
<item>
<title>
<![CDATA[ go-libp2p in 2022 ]]>
</title>
<link>https://libp2p.io/go-libp2p-2022/</link>
<guid>https://libp2p.io/go-libp2p-2022/</guid>
<pubDate>Mon, 13 Feb 2023 00:00:00 GMT</pubDate>
<description>
<![CDATA[ We are excited to share with you all the progress that has been made on go-libp2p in 2022. It has been a year full of exciting new features, code organization, and a growing team of talented contributors. ]]>
</description>
</item>
```

https://libp2p.io/go-libp2p-2022/ should be https://blog.libp2p.io/go-libp2p-2022/

**Comment by marten-seemann** [comment]
It would be great if we had an automated test for that (GitHub Actions cron job). We had this problem before, and this is a failure that's otherwise hard to detect.

**Comment by thiswillbeyourgithub** [comment]
Yeah and the rss can only be mentioned when looking at the webpage's source code, and is wrong it points to https://libp2p.io/index.xml instead of https://blog.libp2p.io/index.xml

This was noticed last year there too: https://github.com/libp2p/website/issues/184


But I can't find the code that causes that issue on that repo either.
